src/views/about/FAQs.vue

```diff
@@ -3,45 +3,76 @@
     <h2 id="faqs">FAQ's</h2>
     <h4>Where can I find the source code?</h4>
     <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat.
+      The source code for all BioThings tools are hosted on the
+      <a href="https://github.com/biothings">BioThings GitHub organization</a>.
+      The source code for the My Geneset API can be found in
+      <a href="https://github.com/biothings/mygeneset.info">this repository</a>.
+      The source code for the My Geneset web app can be found in
+      <a href="https://github.com/biothings/mygeneset.info-website"
+        >this repository</a
+      >.
     </p>
-    <h4>Where can I suggest a feature or more data?</h4>
+    <h4>Where can I suggest a new feature?</h4>
     <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat.
+      If you'd like this website to have new functionality or behave
+      differently,
+      <a href="https://github.com/biothings/mygeneset.info-website/issues"
+        >file an issue here</a
+      >. If you'd like more built-in genesets, additional query functionality,
+      or anything else related to the API,
+      <a href="https://github.com/biothings/mygeneset.info/issues"
+        >file an issue here</a
+      >.
     </p>
-    <h4>What sources do the genes come from?</h4>
+    <h4>Where do the genes and genesets come from?</h4>
     <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat.
-    </p>
-    <h4>How many species are available?</h4>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat.
+      The genes themselves come from the BioThings
+      <a href="https://mygene.info/">My Gene</a> project, which contains
+      millions of genes from thousands of species. The built-in genesets come
+      from various public sources:
     </p>
+    <ul>
+      <li>
+        <a
+          href="https://www.gsea-msigdb.org/gsea/msigdb/genesets.jsp?collection=CP:KEGG"
```

Comment:
It probably makes more sense to link the data sources to their official website. For example, KEGG url is https://www.kegg.jp/.

Reply:
Yes, I don't know the best links for each data source so I was hoping you guys could comment and help me fill in the content. Much of what I've put in this PR is placeholder.

Reply:
@ravila4, correct me if I am wrong:
- CTD (Comparative Toxicogenomics Database): https://ctdbase.org
- DO (Disease Ontology): https://disease-ontology.org
- GO (Gene Ontology): http://geneontology.org
- KEGG (Kyoto Encyclopedia of Genes and Genomes): https://www.kegg.jp
- MSigDB (Molecular Signatures Database): https://www.gsea-msigdb.org/gsea/msigdb
- Reactome: https://reactome.org/
- WikiPathways: https://www.wikipathways.org/

Reply:
Is this all of them (so far)?

Reply:
Yes, that's all of them. Links look right to me.